stdlib/Michelson.ju

```diff
@@ -0,0 +1,140 @@
+mod Michelson =
```

Comment:
We don't need this mod at the top...

Though we will once we have a top level thing I supsect it'll look like

```haskelll
module Michelson where
...
```

Though for now, I'll just use the filename and capitalize the first